### PR TITLE
Upadate runtime to 22.08

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

**Comment by bbhtt** [comment]
I can't find the release versions and date anywhere

File: com.brosix.Brosix.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
                     "type": "archive",
                     "dest": "Brosix",
                     "url": "https://www.brosix.com/downloads/builds/official/Brosix_x64.tar.gz",
-                    "sha256": "2a3fd2fb3f69e23f13d5e7a0dd9cfa2acf117fe06ccce3598b25dbceb622bbb5"
+                    "sha256": "becfde787fb084be98cc581f812c73424361826a8119f303d34268054cd86946"
```
Comment (hfiguiere) [inline com.brosix.Brosix.json]: urgh. this is terrible when tarballs are not versioned.

Comment (bbhtt) [reply com.brosix.Brosix.json]: They unfortunately don't provide any versioned releases or release info.

I had to run it to get the version number and the release date is made up.

Comment (bbhtt) [reply com.brosix.Brosix.json]: Looks like they now nuked the 4.8 build, the current url points to a 4.7.3 build when installed in about dialogue. I updated metainfo accordingly

Comment (hfiguiere) [reply com.brosix.Brosix.json]: unversionned download URL

</rage>

**Comment by bbhtt** [comment]
Seems to run

**Pull request merged**